src/Microsoft.OData.Client/ClientEdmModel.cs

```diff
@@ -104,7 +104,7 @@ public IEdmDirectValueAnnotationsManager DirectValueAnnotationsManager
         /// </summary>
         public IEdmEntityContainer EntityContainer
         {
-            get { throw new NotImplementedException(); }
+            get;

```

Comment:
@ElizabethOkerio Did you figure out why it was unimplemented, and it was still working?

Reply:
I think it was not being used with the `ClientEdmModel`..I could also put it back to how it was since I'm not using it any more.

Reply:
Actually, now that you removed the `throw new NotImplementedException();`, what does it return now that it doesn't have a setter? I think it'd actually be better to leave it the way it was...

Reply:
I left it as it was. Didn't ,make any changes to it. It is not even on the changed files